src/whisper_at_train/run_combined_training.sh

```
@@ -6,50 +6,98 @@
+#SBATCH --error=./log/%j_as_err.txt
 
 set -x
+set -e  # Exit on error
+
 # comment this line if not running on sls cluster
 #. /data/sls/scratch/share-201907/slstoolchainrc
-source /home/taemyung_heo/.cache/pypoetry/virtualenvs/whisper-at-z6hdRBdT-py3.10/bin/activate
+
+# Activate virtual environment
+if [ -f "/home/taemyung_heo/.cache/pypoetry/virtualenvs/whisper-at-z6hdRBdT-py3.10/bin/activate" ]; then
+    source /home/taemyung_heo/.cache/pypoetry/virtualenvs/whisper-at-z6hdRBdT-py3.10/bin/activate
```

Shellcheck (0.10.0)

[info] Not following: /home/taemyung_heo/.cache/pypoetry/virtualenvs/whisper-at-z6hdRBdT-py3.10/bin/activate was not specified as input (see shellcheck -x).

(SC1091)